### ISSUE TITLE
GroupRepresentation's access property should also contain values for viewMembers and manageMembers

### ISSUE THREAD

**Issue by antikalk** [keycloak/keycloak#17344]
### Description

At the moment the `access` property of a `GroupRepresentation` does not contain the fine grained access permissions for `viewMembers` and `manageMembers`. 

### Motivation

This might be necessary for the admin ui to know what is allowed and what not when working with fine-grained auth. With the new properties the ui could decide on when to render the members tab e.g. 

See: https://github.com/keycloak/keycloak/issues/17589

### Details

See `GroupPermissions.getAccess(GroupModel group)`. 

GroupRepresentation atm example:
```json
{
  [...]
  "access": {
    "view": true,
    "manage": false,
    "manageMembership": false
  }
}
```

Should be example:
```json
{
  [...]
  "access": {
    "view": true,
    "manage": false,
    "manageMembership": false,
    "viewMembers": false,
    "manageMembers": false
  }
}
```

**Comment by sschu** [comment]
This could make sense. But since there are costs involved retrieving this data which is only used for the UI, it should only be there optionally IMHO.

**Comment by antikalk** [comment]
@sschu That makes sense and it seems like it is already implemented like this: I just checked whether the `/groups` API does return that `access` property and it does not... Only the `/admin-ui-groups` gets the `access` property in the response.

**Comment by antikalk** [comment]
Closed with #19423
